recipe/meta.yaml

```diff
@@ -22,7 +22,7 @@ source:
   sha256: ededa12ca622baad706ea0a500a358ea51146535466afabd96e558265dc586a2  # [linux64]
   sha256: 7176083a4dad44cb0176771be6efb3775748ad30a39292bf7b4584510f1dd811  # [ppc64le]
   sha256: 4214a0f7b44747c738f2b643be06b2b24826bd1bae6af27f29f3c6dec131bdeb  # [aarch64]
```

Comment:
It seems the checksums are not updated by the bot...

Reply:
Pending CI, I believe I fixed this.